build.rs

```diff
@@ -18,12 +18,14 @@ fn opencv_link() {
                 .filter_map(|entry| entry.ok())
                 .find(|entry| {
                     let file_name = entry.file_name().to_string_lossy().into_owned();
-                    file_name.starts_with("opencv_world") && !file_name.ends_with("d.lib")
+                    (file_name.starts_with("opencv_world") || file_name.starts_with("libopencv_world")) && !file_name.ends_with("d.lib")
                 });
             if let Some(opencv_world) = opencv_world_entry {
-                let opencv_world = opencv_world.path();
+                let opencv_world = opencv_world.file_name();
```

Comment:
This seems complicated enough to justify some comments as clarifications in the code. Maybe just explain it is possible to have `.dll.a` or `.lib`, so we are trim everything until we hit the number in 
version number.

Reply:
I think `opencv_world_without_extension` is kinda self-explaining name. I can add a comment here, but I'm not sure if we need it anyway.

Reply:
Thanks for the tip. I thought my comments are somewhat optional to address so I approved the PR. For future PRs, I will **not** approve unless I will merge immediately.